scripts/or-tools/6.0/script.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+MASON_NAME=or-tools
+MASON_VERSION=6.0
+MASON_LIB_FILE=lib/libortools.${MASON_DYNLIB_SUFFIX}
+
+. ${MASON_DIR}/mason.sh
+
+function mason_load_source {
+    mason_download \
+        https://github.com/google/or-tools/archive/v${MASON_VERSION}.tar.gz \
+        dd78bad745c8416ea27bb66f8162adffb6d4ab24
+
+    mason_extract_tar_gz
+
+    export MASON_BUILD_PATH=${MASON_ROOT}/.build/or-tools-${MASON_VERSION}
+}
+
+function mason_prepare_compile {
+    CCACHE_VERSION=3.3.1
+    CLANG_VERSION=4.0.0
+
+    ${MASON_DIR}/mason install clang++ ${CLANG_VERSION}
+    MASON_CLANG=$(${MASON_DIR}/mason prefix clang++ ${CLANG_VERSION})
+    ${MASON_DIR}/mason install ccache ${CCACHE_VERSION}
+    MASON_CCACHE=$(${MASON_DIR}/mason prefix ccache ${CCACHE_VERSION})
+    ${MASON_DIR}/mason install gflags 2.1.2
+    MASON_GFLAGS=$(${MASON_DIR}/mason prefix gflags 2.1.2)
+    ${MASON_DIR}/mason install protobuf 3.0.0
+    MASON_PROTOBUF=$(${MASON_DIR}/mason prefix protobuf 3.0.0)
+    ${MASON_DIR}/mason install sparsehash 2.0.2
+    MASON_SPARSEHASH=$(${MASON_DIR}/mason prefix sparsehash 2.0.2)
+}
+
+function mason_cflags {
+    echo "-I${MASON_PREFIX}/include"
+}
+
+function mason_ldflags {
+    echo "-L${MASON_PREFIX}/lib -lortools -lz"
+}
+
+function mason_compile {
+    export CXX="${CXX:-${MASON_CLANG}/bin/clang++}"
+    export CC="${CC:-${MASON_CLANG}/bin/clang}"
+
+    # The following patch to the build script disables some of the more useless
+    # and heavyweight parts of the build, like building the automake and autoconf
+    # .info docs with TeXinfo.
+    patch -N -p1 < ${MASON_DIR}/scripts/${MASON_NAME}/${MASON_VERSION}/patch.diff
+
+    export CXXFLAGS="${CXXFLAGS} -fpermissive -02 -g"
```

Comment:
I think the `-02` looks like a type. It should be `ohhh` instead of `zero`. Also the `-fpermissive` can also likely be removed (as I presume the compile error that helped fix would be fixed in 6.0).